gulpfile.js

```diff
@@ -7,9 +7,11 @@ var doxxOptions = {
   },
   src: [
   '!./node_modules/**/*',
+  '!./.publish/**/*',
   '!./test/**/*',
   '!./gulpfile.js',
   '!./local/**/*',
+  '!./docs/**/*',
   './**/*.js',
   './README.md'],
   dest: './docs',
```

Comment:
@e2tha-e this is the line which selected the target location for the final documentation